resources/public/js/orchestrator.js

```diff
@@ -933,7 +933,11 @@ function renderInstanceElement(popoverElement, instance, renderType) {
       contentHtml += '<p><strong>Master</strong></p>';
     }
     if (renderType == "search") {
-      contentHtml += '<p>' + 'Cluster: <a href="' + appUrl('/web/cluster/' + instance.ClusterName) + '">' + instance.ClusterName + '</a>' + '</p>';
+      if (instance.SuggestedClusterAlias) {
+        contentHtml += '<p>' + 'Cluster: <a href="' + appUrl('/web/cluster/' + instance.SuggestedClusterAlias) + '">' + instance.SuggestedClusterAlias + '</a>' + '</p>';
```

Comment:
While the existing code is correct, it is more explicit to strictly use `/web/cluster/alias`.
```suggestion
        contentHtml += '<p>' + 'Cluster: <a href="' + appUrl('/web/cluster/alias/' + instance.SuggestedClusterAlias) + '">' + instance.SuggestedClusterAlias + '</a>' + '</p>';
```